api.yaml

```diff
@@ -811,6 +811,13 @@ components:
       properties:
         version:
           type: integer
+    SchemaState:
```

Comment:
question: are there any consumers relying on this API? Is this breaking change?

Reply:
No. This is endpoint that is known to be only used by people operating the service, so the only thing we possibly need to update is a runbook.

Reply:
One place that needs to change is automation around moving db to specific version for integration tests of other services, but that needs to change after we merge.